sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -12,7 +12,7 @@
 
 from sklearn import datasets
```

Comment:
Is this line still necessary?

Reply:
Yes it's used elsewhere in the test file.